### DISCUSSION TITLE
제페토 개발자 디스코드가 오픈될 예정입니다.

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#1244] [1. Announcements]
안녕하세요, 크리에이터 여러분.

github discussion에서 서로 많은 노하우를 공유해주시고, 좋은 월드를 만들어 주셔서 감사합니다.
운영을 하면서 여러분들께서 원하는 지식들을 보다 쉽게 찾을 수 있고,
마음 맞는 동료도 만날 수 있는 커뮤니티가 되면 좋겠다는 고민을 했었습니다.

그 결과, 글로벌 커뮤니티로의 확장과 함께 디스코드를 오픈하기로 하였습니다. 

현재는 pre-open 상태로 세팅 중에 있으며 9월 중에 정식 오픈할 계획에 있습니다. 

정식 오픈이 되면 github discussions은 read-only로 전환될 예정입니다.
디스코드로의 전환 이후에도 Issues를 통한 버그 제보, Sample 공유는 여전히 github을 사용할 예정이오니
계속해서 많은 이용 부탁 드립니다. 🥰

감사합니다. 

<hr/>

Hello, creators.

Thank you for sharing a lot of know-how in the github discussion and making great world contents.
I was thinking about how to become a community where can you meet colleagues or can find issues more easily.

As a result, we decided to open Discord along with the expansion into the global community.

It is currently being set as pre-open and is planning to officially open in September.

When it is officially opened, github discussions will be converted to read-only.
Even after switching to Discord, we will still use github to report bugs and share samples through Issues.
Please continue to use it a lot.   

thank you

**Comment by Deytooh** [comment]
Oh that's a good news ! But personally, I think it should be better to not set the github discussions as read-only

**Reply by Zepeto-TechEvangelist** [reply]
Please understand that to focus more on Discord.

**Reply by hwisulee** [reply]
This is what I thought. I think read-only is not good way.
Maybe using Discord Bot to send new discussion is better good for me or us.

**Comment by chlguswls32** [comment]
안녕하세요. 

제페토 에반젤리스트님. 먼저 디스코드가 오픈된다는것은 너무 좋은소식이네요.
궁금한점은 깃허브 토론이 read-only로 변경되면 현재와 무엇이 달라지는것인지 궁금합니다.

read-only라는 단어에서 이해되는것처럼, 
질문, 댓글등의 글을 쓸수없게되는것일까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 크리에이터님~
디스코드 오픈 소식을 좋아해주셔서 감사합니다 😄 

깃헙 디스커션이 read-only로 전환 될 경우 새로운 글은 더이상 작성하실 수 없습니다.
대신 댓글은 작성이 가능합니다.

디스코드에서 보다 편리하게 질문-답변을 주고 받으며 
각 이슈에 대한 데이터가 쌓여가는 양질의 글로벌 커뮤니티로 성장시켜 나가기 위함이오니 양해 부탁 드립니다.

감사합니다.

**Comment by ckdlskgkdrhd** [comment]
oh hellll yeeeeee

**Comment by Asipe42** [comment]
디스코드 링크를 부탁드려요!

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 제페토 개발자 디스코드 링크 안내 드립니다. 
https://discord.gg/6Z9dBM7H3e

아래는 로드맵 링크입니다.
https://github.com/orgs/naverz/projects/10

디스코드에서의 활발한 활동 부탁 드립니다 👍 
감사합니다.
